src/SarifWeb/Views/Validation/Index.cshtml

```diff
@@ -510,32 +506,10 @@
                 }
 
                 clearRuleList();
-                setToolbarEnabled(false);
                 setResultPositionText(0, 0);
                 setResultLocationDetails();
             }
 
-            function setToolbarEnabled(enable) {
```

Comment:
>setToolbarEnabled [](http://example.com/codeflow?start=21&length=17)

we won't need to enable/disable. So, i'm removing the function and the places it was calling it.